docker/docker-compose.yaml

```diff
@@ -39,7 +39,48 @@ services:
     container_name: unstract-execution-consumer
     restart: unless-stopped
     entrypoint: .venv/bin/celery
-    command: "-A backend worker --loglevel=info -Q celery,celery_periodic_logs,celery_log_task_queue --autoscale=8,1"
+    command: "-A backend worker --loglevel=info -Q celery --autoscale=${EXECUTION_CONSUMER_AUTOSCALE}"
+    env_file:
+      - ../backend/.env
+    depends_on:
+      - redis
+    environment:
+      - ENVIRONMENT=development
+    labels:
+      - traefik.enable=false
+    volumes:
+      - ./workflow_data:/data
+      - ${TOOL_REGISTRY_CONFIG_SRC_PATH}:/data/tool_registry_config
+
+
+  # Celery worker for managing logs and periodic tasks
+  log-management-worker:
+    image: unstract/backend:${VERSION}
+    container_name: unstract-log-management-worker
+    restart: unless-stopped
+    entrypoint: .venv/bin/celery
+    command: "-A backend worker --loglevel=info -Q celery_periodic_logs,celery_log_task_queue --autoscale=${LOG_MANAGEMENT_WORKER_AUTOSCALE}"
+    env_file:
+      - ../backend/.env
+    depends_on:
+      - redis
+    environment:
+      - ENVIRONMENT=development
+    labels:
+      - traefik.enable=false
+    volumes:
+      - ./workflow_data:/data
+      - ${TOOL_REGISTRY_CONFIG_SRC_PATH}:/data/tool_registry_config
```

Comment:
@muhammad-ali-e why are these volumes present for this log management worker but not the API deployment one?

Reply:
@ali do we really require to attach volume for log management worker?

Reply:
@chandrasekharan-zipstack even for API deployment it is present.

Reply:
It is required for API Deployment. But Yes, It not reqiored for log management worker